### ISSUE TITLE
Issue with conversion of prefLabel containing non-english alphabet characters

### ISSUE THREAD

**Issue by niva83** [sparna-git/skos-play#61]
I am using SKOS play to converted controlled vocab defined in Excel sheet to TTL. 
However I faced an issue of conversion of skos concepts which preferred labels contain non-English characters. 

Example, a concept in `skos:prefLabel` column in Excel has value set to "Côte d'Ivoire".
When I run SKOS Play and convert this Excel sheet, the corresponding RDF `preLabel` is converted to "CÃ´te d'Ivoire": 
```TTL
vocab:CI a skos:Concept;
  dct:source <https://en.wikipedia.org/wiki/ISO_3166-1>;
  skos:broader vocab:ISO3166-1;
  skos:inScheme <http://cog.link/vocab/enel-base>;
  skos:notation "384"^^vocab:iso3166-1-numeric, "CI"^^vocab:iso3166-1-alpha-2, "CIV"^^vocab:iso3166-1-alpha-3;
  skos:prefLabel "CÃ´te d'Ivoire" .
```

I tried specifying language tag for `prefLable`, however that did not help.

**Comment by niva83** [comment]
However, there are no problems if conversion is executed using cli tool `xls2rdf` v.2.1.1

**Comment by tfrancart** [comment]
Hello Nikola can you attach a test file here ?
This surprises me, being French I have converted tons of files with accented non-ASCII characters, and never noticed any issue.

**Comment by nicholas512** [comment]
I ran into a similar issue with the attached file.  `é` becomes `Ã©`.  Also `°` becomes `Â°`
[SKOSplay-convert.xlsx](https://github.com/sparna-git/skos-play/files/7738336/SKOSplay-convert.xlsx)

I think its an encoding issue somewhere in excel but I'm not sure where.

**Comment by tfrancart** [comment]
Hello @nicholas512 thanks for the test file. I cannot reproduce the issue though (Linux+Firefox). What I _can_ reproduce is that, when the output turtle file is displayed in the browser in Windows+Edge, the accented characters are shown as you say. But when the output is saved to a file, and opened in an editor, the content is displayed correctly.

Note that a Turtle file is _always_ in UTF-8, as per the spec. So what happens is that the output is in UTF-8, but the browser does not know it and tries to display it in Windows encoding.

Try to get the output in RDF/XML to see if it changes something.

What is your browser / OS config ?

What could be happening is that the HTTP answer is missing a proper response header to indicate the answer encoding. I will investigate that.

**Comment by tfrancart** [comment]
I have added an extra response header with proper encoding, and the results in the browser is displayed correctly. Can you try it out and let me know if it is OK for you ? if yes, please close the issue. Thanks

**Comment by niva83** [comment]
@tfrancart first of all sorry for not responding. Second, I see that the issue has been fixed, and third here is an example of [my XLSX](https://docs.google.com/spreadsheets/d/1oaEUkbzB4hw6wW4SMf8EiPdKVqZuqvVm/edit?usp=sharing&ouid=113182180159189725200&rtpof=true&sd=true).

**Comment by nicholas512** [comment]
Thanks @tfrancart for the very quick reply.  I tried my file again and can confirm that the characters now show up correctly in the output.

> What I can reproduce is that, when the output turtle file is displayed in the browser in Windows+Edge, the accented characters are shown as you say. But when the output is saved to a file, and opened in an editor, the content is displayed correctly.

Interesting.  I got the weird characters in both the HTML response (Windows + Chrome) and also when I opened it in Notepad++.  

> What is your browser / OS config ?

I am on Windows 10 (10.0.19043)/ Chrome (96.0.4664.93 (Official Build) (64-bit)).

I'm not able to close the issue (maybe @niva83 is able to) but as far as I'm concerned this issue has been resolved.
